// [[Reason: Invalid computed property can only be detected by TSC]] ////

//// [tests/cases/conformance/parser/ecmascript5/Symbols/parserES5SymbolProperty1.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,8 +1,9 @@
 
 
 //// [parserES5SymbolProperty1.d.ts]
 interface I {
+    [Symbol.iterator]: string;
 }
 
 /// [Errors] ////
 
